strawberry/field.py

```diff
@@ -76,17 +75,11 @@ def __init__(
 
         # kw_only was added to python 3.10 and it is required
         if sys.version_info >= (3, 10):
-            kwargs["kw_only"] = dataclasses.MISSING
+            kwargs["kw_only"] = True
```

Comment:
@paulo-raca does this make sense to you? I think kw_only should be True by default, no?

Reply:
Either works. MISSING` will inherit it its value from the @dataclass, which is also True

Reply:
ah, nice, maybe I'll should default to missing then